src/Stratis.Bitcoin.Features.Consensus/ConsensusOptions.cs

```diff
@@ -27,13 +21,8 @@ public class PosConsensusOptions : PowConsensusOptions
         /// </summary>
         public PosConsensusOptions()
         {
-            this.MaxMoney = long.MaxValue;
-            this.CoinbaseMaturity = 50;
-
-            this.ProofOfWorkReward = Money.Coins(4);
+            this.MaxMoney = long.MaxValue;           
```

Comment:
I think this.MaxReorgLength can also go in consensus its not necessarily a pos thing.
Well it is but it could potentially be used on a pos netwrok